app/controllers/users_controller.rb

```diff
@@ -42,6 +42,14 @@ def destroy
 
   private
 
+  def upload_picture(user, profile_pic)
+    if profile_pic
```

Comment:
```suggestion
    return unless profile_pic
```

This way you don't have to worry about it in the rest of the method